tcl/html.tcl

```diff
@@ -536,6 +538,114 @@ proc qc::safe_attributes_check {node} {
     return true
 }
 
+
+proc qc::html_error_report {text} {
```

Comment:
Was considering prefixing with `safe` to recognise association with a safe_html data type.

Some ideas: 

* safe_html_error_report
* safe_html_violations
* safe_html_errors
* safe_html_report